wand-watcher.lic

```diff
@@ -0,0 +1,153 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#wand-watcher
+=end
+
+no_kill_all
+no_pause_all
+
+custom_require.call(%w[common common-items])
+
+class WandWatcher
+  include DRC
+  include DRCI
+
+  def initialize
+    settings = get_settings();
+    @wand_list = settings.wands
+    @queued_wands = []
+    @no_use_scripts = settings.wand_watcher_no_use_scripts || []
+    @startup_delay = settings.wand_watcher_startup_delay || 10
+    @passive_delay = settings.wand_watcher_passive_delay || 60
+    @activation_failure_messages = [/^The .* remains inert/]
```

Comment:
Proposing list of rooms where not to try and use the wands, such as banks and vaults or other areas where magical devices are restricted.
```suggestion
    @activation_failure_messages = [/^The .* remains inert/]
    @activation_blocked_messages = ["This is not a good place for that.", "You shouldn't disrupt the area", "Something in the area is interfering"]
```